fs/operations/rc.go

```diff
@@ -253,7 +254,7 @@ func rcSingleCommand(ctx context.Context, in rc.Params, name string, noRemote bo
 					return nil, err
 				}
 				if p.FileName() != "" {
-					obj, err := Rcat(ctx, f, p.FileName(), p, time.Now())
+					obj, err := Rcat(ctx, f, filepath.Join(remote, p.FileName()), p, time.Now())
```

Comment:
That should be `path.Join` not `filepath.Join`. `filepath` is for OS native strings so will have \ instead of / on Windows.

Reply:
Ahh I see. Will update that